streaming/BUILD.bazel

```diff
@@ -143,6 +152,60 @@ cc_library(
     }),
 )
 
+cc_library(
+    name = "streaming_channel",
+    srcs = glob(["src/channel/*.cc"]),
+    hdrs = glob(["src/channel/*.h"]),
+    includes = ["src/channel/"],
+    deps = [
+        ":streaming_util",
+        ":streaming_queue",
+        ":streaming_message",
+        ":streaming_ring_buffer",
+        ":streaming_common",
+    ],
+    copts = COPTS,
+    visibility = ["//visibility:public"],
+)
+
+cc_library(
+    name = "streaming_reliability",
+    srcs = glob([ "src/reliability/*.cc" ]),
+    hdrs = glob([ "src/reliability/*.h"]),
+    includes = ["src/", "src/reliability/"],
+    deps = [
+        ":streaming_util",
+        ":streaming_channel",
+        ":streaming_message",
+    ],
+    copts = COPTS,
+    visibility = ["//visibility:public"],
+)
+
+cc_library(
+    name = "streaming_ring_buffer",
```

Comment:
Why make ringbuffer as separated library?

Reply:
To avoid cycling imports.